app/assets/javascripts/sequence_show.js

```diff
@@ -42,6 +45,26 @@ function init_sequence_show(data) {
         });
     }
 
+    function initD3TreeView(data, page) {
+      $("#treeview-reset").click(function() {
+        $("#d3TreeView").treeview('reset');
+      });
+        $(page).treeview({
+          data: data,
+          width: 916,
+          height: 600,
+        });
+      nodeFillColor = function nodeFillColor(d) {
```

Comment:
what are these functions doing here?


_[Original comment](https://github.ugent.be/unipept/unipept/pull/528#discussion_r3464) by @bmesuere on Mon Nov 09 2015 at 20:53._